examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -169,7 +169,12 @@ end
 function implicit_tendency!(Yₜ, Y, p, t)
     @nvtx "implicit tendency" color = colorant"yellow" begin
         Fields.bycolumn(axes(Y.c)) do colidx
-            CA.implicit_tendency!(Yₜ, Y, p, t, colidx)
+            CA.implicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx)
+
+            if p.tendency_knobs.has_turbconv
+                parent(Yₜ.c.turbconv[colidx]) .= zero(eltype(Yₜ))
+                parent(Yₜ.f.turbconv[colidx]) .= zero(eltype(Yₜ))
```

Comment:
Where are the non-turbconv tendencies zeroed out? Shouldn't we add this bit of code there?

Reply:
They aren't zeroed out anywhere. They are just set in `implicit_vertical_advection_tendency`. At some point, we should probably go through the tendencies and make them all "incrementing" rather than "setting", since that will allow us to change the order of tendency evaluations or to temporarily disable certain tendencies.